test/IRGen/builtin_isConcrete.sil

```diff
@@ -0,0 +1,27 @@
+// RUN: %target-swift-frontend -emit-ir -parse-sil %s -module-name Swift -parse-stdlib | %FileCheck %s
+
+import Builtin
+
+struct MyInt {
+  var value: Builtin.Int32
+}
+
+// CHECK-LABEL: define swiftcc i1 @isConcrete_true() {{.*}} {
```

Comment:
```
21:53:56 FAIL: Swift(linux-x86_64) :: IRGen/builtin_isConcrete.sil (11076 of 12152)
21:53:56 ******************** TEST 'Swift(linux-x86_64) :: IRGen/builtin_isConcrete.sil' FAILED ********************
21:53:56 Script:
21:53:56 --
21:53:56 : 'RUN: at line 1';   /home/buildnode/jenkins/workspace/swift-PR-Linux-smoke-test/branch-master/buildbot_linux/swift-linux-x86_64/bin/swift -frontend -target x86_64-unknown-linux-gnu  -module-cache-path '/home/buildnode/jenkins/workspace/swift-PR-Linux-smoke-test/branch-master/buildbot_linux/swift-linux-x86_64/swift-test-results/x86_64-unknown-linux-gnu/clang-module-cache' -swift-version 4   -typo-correction-limit 10  -emit-ir -parse-sil /home/buildnode/jenkins/workspace/swift-PR-Linux-smoke-test/branch-master/swift/test/IRGen/builtin_isConcrete.sil -module-name Swift -parse-stdlib | '/usr/bin/python' '/home/buildnode/jenkins/workspace/swift-PR-Linux-smoke-test/branch-master/swift/utils/PathSanitizingFileCheck' --sanitize BUILD_DIR='/home/buildnode/jenkins/workspace/swift-PR-Linux-smoke-test/branch-master/buildbot_linux/swift-linux-x86_64' --sanitize SOURCE_DIR='/home/buildnode/jenkins/workspace/swift-PR-Linux-smoke-test/branch-master/swift' --use-filecheck '/home/buildnode/jenkins/workspace/swift-PR-Linux-smoke-test/branch-master/buildbot_linux/llvm-linux-x86_64/bin/FileCheck'  /home/buildnode/jenkins/workspace/swift-PR-Linux-smoke-test/branch-master/swift/test/IRGen/builtin_isConcrete.sil
21:53:56 --
21:53:56 Exit Code: 1
21:53:56 
21:53:56 Command Output (stderr):
21:53:56 --
21:53:56 
/home/buildnode/jenkins/workspace/swift-PR-Linux-smoke-test/branch-master/swift/test/IRGen/builtin_isConcrete.sil:9:17: error: CHECK-LABEL: expected string not found in input
21:53:56 // CHECK-LABEL: define swiftcc i1 @isConcrete_true() {{.*}} {
21:53:56                 ^
21:53:56 <stdin>:1:1: note: scanning from here
21:53:56 ; ModuleID = '-'
21:53:56 ^
21:53:56 <stdin>:6:12: note: possible intended match here
21:53:56 module asm ".section .swift1_autolink_entries,\220x80000000\22"
21:53:56            ^
```

@stephentyrone @gottesmm @atrick any idea as to why IR isn't being emitted in the test suite?

Reply:
I'm guessing the issue was that I wasn't placing visibility specifiers in the checked function declarations.